docusaurus/docs/operate/morse_migration/11_testing.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Migration Testing
+sidebar_position: 11
+---
+
+## Table Of Contents
+
+- [Background](#background)
+- [1. Retrieve a Morse TestNet Snapshot](#1-retrieve-a-morse-testnet-snapshot)
+- [2. Export Morse TestNet Export State](#2-export-morse-testnet-export-state)
+- [3. Transform & Merge Morse Exports to a Canonical Account State Import Message](#3-transform--merge-morse-exports-to-a-canonical-account-state-import-message)
+
+## Background
+
+For testing the migration process end-to-end, _on a live and public network and with real stakeholders_, the Morse state
+is initially imported into Shannon TestNet(s).
+
+In order to maximize both developer convenience and peace of mind, the Morse states which are imported into Shannon
+TestNet(s) are **a merge of both a Morse MainNet and a Morse TestNet state exports**.
+
+## 1. Retrieve a Morse TestNet Snapshot
+
+Morse state exports are derived from snapshots. Since Morse TestNet snapshots are not automated, like Morse MainNet
+snapshots are, Morse TestNet snapshots are taken manually and distributed via STORJ network.
+Links are provided here for convenience:
+
+| Snapshot                                                                                                                                                      | Height | Date       | Size   | 
+|---------------------------------------------------------------------------------------------------------------------------------------------------------------|--------|------------|--------|
+| [morse-tesnet-176681-2025-05-07.txz](https://link.storjshare.io/raw/jwndx6se4o6tdwpeqhxm7imiam6a/pocket-network-snapshots/morse-tesnet-176681-2025-05-07.txz) | 176681 | 2025-05-07 | 7.37GB |
+
+In order to generate or reproduce the canonical merged Morse export state, the snapshot heights for used then
+reproducing MUST match what those used when generating the canonical state.
+See the table in [Migration Artifacts](https://github.com/pokt-network/poktroll/tree/main/tools/scripts/migration) to
+ensure the correctness of snapshot heights.
+
+Export the snapshot into a new directory on your local machine.
+
+```bash
+mkdir -p $HOME/morse-testnet-snapshot
+# 1. Untar the snapshot file
+tar -xvf <testnet-snapshot-file>.tar -C $HOME/morse-testnet-snapshot
+# 2. Change directory to the extracted snapshot folder
+cd $HOME/morse-testnet-snapshot
+```
+
+## 2. Export Morse TestNet Export State
```

Comment:
I would rather merge this into the previous section and provide a single copy pastable code to both retrieving and exporting the state.

It would also be a good opportunity to automate the extraction of the `height` and `date` from the downloaded file.

The idea would be to endup with an export with a single copy past (if possible)

Reply:
This division between extracting the snapshot tarball and exporting the state from that mirrors the existing steps outlined in the [State Transfer Playbook](https://dev.poktroll.com/operate/morse_migration/state_transfer_playbook).

I think it's prudent to keep them separate, in case an error occurs in-between (e.g. out of disk space :sweat_smile:).

Reply:
Neat idea about trying to parse the file name, I'd rather not experiment with any more bash than is  absolutely necessary. :sweat_smile: